website/README.md

```diff
@@ -10,6 +10,8 @@ $ yarn
 
 ### Local Development
 
+Make a sibling copy of `sidebars.template.json`, named `sidebars.json`. This file is gitignored, and will setup the local dev sidebar.
```

Comment:
Is this change intentional.... Doesn't really look like it matters either way....

Reply:
The README is missing an explanation of how to make the website startup (the sidebar has been busted for a while; looks like someone finally fixed it, though didn't mention the manual steps necessary to make it work locally).